test/javascript/components/interchange.js

```diff
@@ -170,27 +170,19 @@ describe('Interchange', function() {
     it('calls reflow on viewport size change once', function(done) {
       $html = $(generateTemplate('image')).appendTo('body');
       plugin = new Foundation.Interchange($html, {});
-      setTimeout(function() {
+      setTimeout(function () {
         Foundation.IHearYou();
       }, 1);
       let spy = sinon.spy(plugin, '_reflow');
 
-      setTimeout(function() {
+      setTimeout(function () {
         $(window).trigger('resize');
-      }, 5);
-
-      setTimeout(function() {
-        $(window).trigger('resize');
-      }, 10);
-
-      setTimeout(function() {
-        $(window).trigger('resize');
-      }, 20);
```

Comment:
This break the purpose of the test itself. `calls reflow on viewport size change once` means that with multiple `resize`, `_reflow` is called only once within a given delay (10ms by default for MutationObserver debounce). We cannot test this behavior without triggering `resize` once.

So we actually broke these test before with switching to delays over 10ms. We should triggers several `resize` in 10ms and expect reflow to be called once.

Also, this should not be in this PR.

Reply:
Right. But not fixing this in this PR will produce a failing Travis build.

What do you recommend because of this issue with reflow and the timing issues on the Travis containers?

Reply:
> Right. But not fixing this in this PR will produce a failing Travis build.

Are you sure ? Tests runs after `tests: always call blur() for asynchronous calls`. Do we need both commits for tests to runs ?

I would recommend to actually test what we want to test and expect it to fail if the component does not work properly: trigger `resize` several times in 10ms and check after that if `_reflow` was called once.

This _shoud_ work. If it doesn't, Interchange has a bug. I'll take a look at this later.

Reply:
> I'll take a look at this later.

This would be really helpful, thanks.